index.js

```diff
@@ -75,8 +75,12 @@ const Minesweeper = {
 		const y = this.cursorY
 		const isFlagged = !this.flagged[x][y]
 		this.flagged[x][y] = isFlagged
-		if (this.mines[x][y]) this.flaggedMines += isFlagged ? 1 : -1
-		if (this.flaggedMines >= this.nrOfMines) return this.submit()
+		if (this.mines[x][y]) {
+			this.flaggedMines += isFlagged ? 1 : -1
+		} else {
+			this.flaggedTiles += isFlagged ? 1 : -1
```

Comment:
`flaggedTiles` here means "flagged tiles that are not mines", right? Would prefer to have it count `flaggedTiles`, so we can check `flaggedTiles <= flaggedMines`.